runelite-client/src/main/java/net/runelite/client/plugins/nightmarezone/NightmareZonePlugin.java

```diff
@@ -193,6 +213,32 @@ private void checkAbsorption()
 		}
 	}
 
+	private int calculatePointsPerHour()
+	{
+		Instant now = Instant.now();
+		final int currentPoints = client.getVar(Varbits.NMZ_POINTS);
+
+		if (nmzSessionStartTime == null)
+		{
+			nmzSessionStartTime = now;
+		}
+
+		Duration timeSinceStart = Duration.between(nmzSessionStartTime, now);
+
+		if (!timeSinceStart.isZero())
+		{
+			return (int) ((double) currentPoints * (double) HOUR.toMillis() / (double) timeSinceStart.toMillis());
+		}
+
+		return pointsPerHour;
```

Comment:
This should probably be 0? Or -1 and then don't show it in the overlay?

Reply:
Changed it to 0. I think that restricting the overlay based on -1 there would be unnecessary since it would be -1 only during the first tick of a NMZ session.